CMakeLists.txt

```diff
@@ -82,6 +82,14 @@ if(WIN32)
   set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY_MINSIZEREL ${WINDOWS_APPLICATION_OUT_DIRECTORY})
 endif()
 
+# FreeBSD support
+
+# link BSD threading
```

Comment:
Could you please move the comment inside the `#if`? Also, starting the comment with a capital letter IMO looks better (like in other places in this file) ;)